app/src/main/java/org/greenstand/android/TreeTracker/messages/announcementmessage/AnnouncementScreen.kt

```diff
@@ -0,0 +1,249 @@
+                            .fillMaxSize()
+                            .padding(
+                                top = 10.dp,
+                                bottom = 10.dp
+                            )
+                            .background(color = AppColors.MediumGray, shape = CircleShape),
+                        contentAlignment = Alignment.Center
+                    ) {
+                        Text(
+                            text = stringResource(id = R.string.admin_placeholder).uppercase(),
+                            color = CustomTheme.textColors.lightText,
+                            fontWeight = FontWeight.Bold,
+                            style = CustomTheme.typography.regular,
+                        )
+                    }
+                },
+                centerAction = {
+                    Image(
+                        painter = painterResource(id = R.drawable.chat_icon),
+                        contentDescription = null,
+                        modifier = Modifier.align(Alignment.Center)
+                    )
+                },
+                rightAction = {
+                    state.currentUser?.photoPath?.let { imagePath ->
+                        LocalImage(
+                            modifier = Modifier
+                                .align(Alignment.Center)
+                                .padding(bottom = 5.dp, top = 5.dp)
+                                .aspectRatio(1.0f)
+                                .size(100.dp)
+                                .clip(CircleShape),
+                            imagePath = imagePath,
+                            contentScale = ContentScale.Crop
+                        )
+                    }
+                }
+            )
+        },
+        bottomBar = {
+            ActionBar(
+                leftAction = {
+                    ArrowButton(
+                        isLeft = true,
+                        colors = AppButtonColors.MessagePurple,
+                        onClick = {
+                            navController.popBackStack()
+                        }
+                    )
+                }
+            )
+        }
+    ) {
+        Column(
+            Modifier
+                .padding(top = 4.dp, start = 4.dp, end = 4.dp, bottom = 80.dp)
+                .fillMaxSize()
+                .nestedScroll(scrollBehavior.nestedScrollConnection)
+        ) {
+            Messages(
+                state = state,
+                modifier = Modifier.weight(1f),
+                scrollState = scrollState,
+                viewModel = viewModel
+            )
+            if (state.showNoInternetDialog) {
+                NoInternetDialog(viewModel = viewModel)
+            }
+        }
+    }
+}
+
+@Composable
+fun Messages(
+    state: AnnouncementState,
+    scrollState: LazyListState,
+    modifier: Modifier = Modifier,
+    viewModel: AnnouncementViewModel
+) {
+    Box(modifier = modifier) {
+        LazyColumn(
```

Comment:
if this screen only ever shows one announcement at a time, there is no need for a scrolling list

Reply:
however the single announcement message could be quite long, so it does need to scroll.